docs/docsite/rst/user_guide/become.rst

```diff
@@ -208,6 +208,32 @@ or '/bin/chmod' as the allowed commands this will fail with ansible as those
 paths won't match with the temporary file that ansible creates to run the
 module.
 
+Environment variables populated by pam_systemd
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The default methods used by ``become`` use the default PAM login modules, but
```

Comment:
This should be sure to mention that it's specific to Linux systems that use `systemd`, Ansible supports Linux distros of  varying init systems as well as other *nix flavors, but otherwise this looks great. Thanks!

Reply:
I rephrased the beginning to make sure people understand this is specific to a Linux+systemd setup.